docs/api.md

```diff
@@ -292,6 +292,11 @@ content-server page.
     signed in email address will be used as the default.
   - If `action` is `force_auth`, the user is unable to modify the email
     address and is unable to sign up if the address is not registered.
+- `keys`: Optional. Boolean setting, set this if relier will need derive relier keys during the signup / signin flow.
```

Comment:
`derive`=>`derived`. `during the signup / signin flow` may be implicit and not needed?